post-scan-actions/gcp-python-promote-or-quarantine/main.tf

```diff
@@ -2,14 +2,14 @@ terraform {
   required_providers {
     google = {
       source = "hashicorp/google"
-      version = "~> 4.0.0"
+      version = "= 4.0.0"
```

Comment:
The `~>` operator fix the version to update the minor version only, so I think we can keep it since we found the root cause is not the version.